client/landing/jetpack-cloud/components/sidebar/style.scss

```diff
@@ -0,0 +1,48 @@
+// Menu links
+.sidebar__menu-link {
+	padding-top: 14px;
+	padding-bottom: 14px;
+	font-weight: 600;
+
+	.selected &,
+	.selected &:hover {
+		background-color: rgba( var( --color-sidebar-menu-selected-background-rgb ), 0.12 );
+	}
+}
+
+// Special case for the "Manage site" menu item icon
+.sidebar__menu-item-back .material-icon {
+	transform: scaleX( -1 );
```

Comment:
The designs on the sidebar itself have since changed so we can use the navigation arrow back instead: https://material.io/resources/icons/?icon=arrow_back&style=baseline

Not super urgent, so it could be added in a separate PR.

Reply:
I was happy to get rid of this hacky "back" icon in 2e4f119 🙂

Reply:
Amazing! Thanks, @delawski